frontend/src/components/landingPage/MoreInformation.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div id="moreInfoPage">
+    <app-header></app-header>
+    <h5></h5>
+    <div id="moreInfoDiv">
+      <div id="moreInfoTextDiv">
+        <p v-if="this.receivedId != this.errorStr">ID : {{ receivedId }}</p>
+        <p v-if="this.groupID != this.errorStr">Group ID : {{ groupID }}</p>
+        <p v-if="this.artifactID != this.errorStr">Artifact ID : {{ artifactID }}</p>
+        <p v-if="this.discovered != this.errorStr">Discovered : {{ discovered }}</p>
+        <p v-if="this.source != this.errorStr">Source : {{ source }}</p>
+        <p v-if="this.version != this.errorStr">Version : {{ version }}</p>
+      </div>
+    </div>
+    <div id="moreInfoTableDiv">
+      <v-data-table :headers="headers" :items="metrics" class="elevation-1"></v-data-table>
+    </div>
+    <app-footer></app-footer>
+  </div>
+</template>
+<script>
+import Header from "../Layout/Header";
+import Footer from "../Layout/Footer";
+import { eventBus } from "../../main";
+
+export default {
+  name: "moreInformation",
+  data() {
+    return {
+      receivedId: String,
+      artifactID: String,
+      groupID: String,
+      discovered: String,
+      source: String,
+      version: String,
+      errorStr: "function String() { [native code] }",
+      metricKeys: [],
+      metricValues: [],
+      metrics: [],
+      headers: [
+        { text: "Metric Name", align: "center", sortable: "true", value: "id" },
+        {
+          text: "Metric Value",
+          align: "center",
+          sortable: "true",
+          value: "values"
+        }
+      ]
+    };
+  },
+  watch : function() {
+    setTimeout(() => {
+      this.startRetrieve();
+    }, 100);
+  },
+  created: function() {
+    eventBus.$on("moreInfoEvent", data => {
+      this.receivedId = data;
+    }),
+    setTimeout(() => {
+      this.startRetrieve();
+    }, 100);
+  },
+  components: {
+    "app-header": Header,
+    "app-footer": Footer
+  },
+  methods: {
+    startRetrieve() {
+      var vm = this;
+      this.$http
+        .get("retrieve/" + vm.receivedId)
+        .then(response => {
+          return response.json();
+        })
+        .then(data => {
+          (vm.groupID = data.messages[0].metadata.groupId),
```

Comment:
You can make your life a bit easier here in the future by just using the metadata object from the result.

Reply:
This was done to have local variables, in case any manipulations are done for the same. However, will keep this idea in mind during further development.